nautobot/docs/development/core/index.md

```diff
@@ -0,0 +1,322 @@
+
+* Good feature requests are very narrowly defined. Be sure to thoroughly
+  describe the functionality and data model(s) being proposed. The more effort you put into writing a feature request, the better its chance is of being
+  implemented. Overly broad feature requests will be closed.
+
+* When submitting a feature request on GitHub, be sure to include all
+  information requested by the issue template, including:
+    * A detailed description of the proposed functionality
+    * A use case for the feature; who would use it and what value it would add to Nautobot
+    * A rough description of changes necessary to the database schema (if applicable)
+    * Any third-party libraries or other resources which would be involved
+    * Please avoid prepending any sort of tag (e.g. "[Feature]") to the issue title.
+
+The issue will be reviewed by a moderator after submission and the appropriate
+labels will be applied for categorization.
+
+Feature requests will follow our published workflow from inbound triage to ultimate terminal state, whether accepted or closed:
+
+![Feature Request Workflow State Diagram](../../media/development/index_feature_request_wf.jpg)
+
+### Submitting Pull Requests
+
+* If you're interested in contributing to Nautobot, be sure to check out our
+  [getting started](getting-started.md)
+  documentation for tips on setting up your development environment.
+
+* It is recommended to open an issue **before** starting work on a pull request, and discuss your idea with the Nautobot maintainers before beginning work. This will help prevent wasting time on something that we might not be able to implement. When suggesting a new feature, also make sure it won't conflict with any work that's already in progress.
+
+* Once you've opened or identified an issue you'd like to work on, ask that it
+  be assigned to you so that others are aware it's being worked on. A maintainer
+  will then mark the issue as "accepted."
+
+* If you followed the project guidelines, have ample tests, code quality, you will first be acknowledged for your work. So, thank you in advance! After that, the PR will be quickly reviewed to ensure that it makes sense as a contribution to the project, and to gauge the work effort or issues with merging into _current_. If the effort required by the core team isn’t trivial, it’ll likely still be a few weeks before it gets thoroughly reviewed and merged, thus it won't be uncommon to move it to _near term_ with a `near-term` label. It will just depend on the current backlog.
+
+* All code submissions should meet the following criteria (CI will enforce
+  these checks):
+    * Python syntax is valid
+    * All unit tests pass successfully
+    * PEP 8 compliance is enforced, with the exception that lines may be
+      greater than 80 characters in length
+    * At least one [changelog fragment](#creating-changelog-fragments) has
+      been included in the feature branch
```

Comment:
```suggestion
* If you're interested in contributing to Nautobot, be sure to check out our [getting started](getting-started.md) documentation for tips on setting up your development environment.

* It is recommended to open an issue **before** starting work on a pull request, and discuss your idea with the Nautobot maintainers before beginning work. This will help prevent wasting time on something that we might not be able to implement. When suggesting a new feature, also make sure it won't conflict with any work that's already in progress.

* Once you've opened or identified an issue you'd like to work on, ask that it be assigned to you so that others are aware it's being worked on. A maintainer will then mark the issue as "accepted."

* If you followed the project guidelines, have ample tests, code quality, you will first be acknowledged for your work. So, thank you in advance! After that, the PR will be quickly reviewed to ensure that it makes sense as a contribution to the project, and to gauge the work effort or issues with merging into _current_. If the effort required by the core team isn’t trivial, it’ll likely still be a few weeks before it gets thoroughly reviewed and merged, thus it won't be uncommon to move it to _near term_ with a `near-term` label. It will just depend on the current backlog.

* All code submissions should meet the following criteria (CI will enforce these checks):
    * Python syntax is valid
    * All unit tests pass successfully
    * PEP 8 compliance is enforced, with the exception that lines may be greater than 80 characters in length
    * At least one [changelog fragment](#creating-changelog-fragments) has been included in the feature branch
```